src/components/SlateEditor/plugins/embed/FigureButtons.tsx

```diff
@@ -35,7 +34,8 @@ const StyledFigureButtons = styled('div')`
   top: 0;
   z-index: 1;
 
-  ${p => p.align !== 'left' && p.align !== 'right' && centerAdjustedStyle}
+  ${(p: any) =>
```

Comment:
hvorfor har du en type her som er any men ikke på de to under? og går det an å finne ut hvilken type i stedenfor å bruke any? :)

Reply:
Jo riktig! Når jeg drev å testa noe fort så satt jeg den til any og glemte å gjøre det ordentlig i etterkant. Når jeg satt typen bare på den ene variabelen så stoppet typescript å klage på typen til de andre variabelene, derfor satt jeg ikke noe type på dem (tror kanskje at siden variabelene heter det samme så klarte typescript å komme fram at de skal ha samme typen og sluttet å klage, men det er bare min gjett). Skal fikse any på de typene her!